i18n/scripts/extract-keys.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable  */
+// disable-translation
+import fs from 'fs'
+import glob from 'glob'
+import path from 'path'
+import { str as crc32 } from 'crc-32'
+import { DISABLE_TRANSLATION, IGNORE_GLOB_PATTERNS, TRANSLATABLE_BASE_PATH, TRANSLATABLE_GLOB_PATTERNS } from '../constants'
```

Comment:
are those constants used anywhere else? If not, can they be just moved here for readability and easy access? Otherwise its fine we can keep them seperate